tests/jballerina-unit-test/src/test/java/org/ballerinalang/test/functions/FunctionWithIncludedRecordParam.java

```diff
@@ -185,4 +185,9 @@ public void testFunctionOfFunctionTypedParamWithIncludedRecordParam21() {
     public void testFunctionOfFunctionTypedParamWithIncludedRecordParam22() {
         BRunUtil.invoke(result, "testFunctionOfFunctionTypedParamWithIncludedRecordParam22");
     }
+
+    @Test
+    public void testFunctionOfFunctionTypedParamWithIncludedRecordParam23() {
+        BRunUtil.invoke(result, "testFunctionOfFunctionTypedParamWithIncludedRecordParam23");
```

Comment:
Are these names correct? IMO, it implies there's a param of a function type.

Reply:
Change as `functionWithIncludedRecordParameters`.